src/snmalloc/override/new.cc

```diff
@@ -71,3 +71,45 @@ void operator delete[](void* p, std::nothrow_t&)
 {
   ThreadAlloc::get().dealloc(p);
 }
+
+void* operator new(size_t size, std::align_val_t)
```

Comment:
Discarding the alignment requirement seems dubious?  While it's generally true that `snmalloc` prefer to align things much more strongly than generally required, it's still possible to ask for yet stronger alignment.  I think the correct thing to do is to use https://github.com/microsoft/snmalloc/blob/d0df2d028a45131e5b85dce873c0bcc572f6a57d/src/mem/sizeclasstable.h#L131 as done with https://github.com/microsoft/snmalloc/blob/d0df2d028a45131e5b85dce873c0bcc572f6a57d/src/override/malloc.cc#L158-L174
I do not know to what extent those checks are also required by C++ .